plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherExtension.kt

```diff
@@ -23,6 +23,21 @@ open class PlayPublisherExtension : AccountConfig by PlayAccountConfigExtension(
             }
         }
 
+    @get:Internal("Backing property for CLI option")
+    internal var _fromTrack = TrackType.INTERNAL
+    /**
+     * Specify the track for your app that will be modified. May be one of internal, alpha, beta, rollout,
+     * or production. Default is internal.
+     */
+    @get:Input
+    internal var fromTrack
```

Comment:
I think that this should be entirely `internal` and the modification should require the CLI option.

Reply:
Why do we need a `fromTrack` at all? It makes sense to me that you'd want to shift the track above your target to the left. So say the target is `beta`, we'd move `alpha` into beta, or if it isn't available, `internal`. Another example: our target is `production` so we take `beta` and move it over (or `alpha` if it isn't available and so on). That way we mirror behavior from the Play Console too.

Reply:
You don't have to use all of the tracks, and you can skip tracks, and have releases in multiple tracks. I may want to promote everything from Alpha to Production, and Internal to Beta (Which is SOP on the Play Console), versus having to march through each of those steps with subsequent commands.

Reply:
For the sake of clarity, I will say that I initially considering using `toTrack` instead of `fromTrack`, because the functionality on the Play Store is: 
"Pick an exiting TrackRelease"->"Choose what Track to promote it to"
Which would _normally_ mean a user would be using their default from their normal gradle script as the start track, but that would mean modifying the `updateTracks` method, and it seemed better to have this specific override.

Reply:
Hmmm, sounds fair